scripts/developer_tools/j_work_on_pr.py

```
@@ -0,0 +1,197 @@
+        if head_repo is None:
+            print(
+                "Error: PR head repository is inaccessible (likely deleted fork); cannot check out.",
+                file=sys.stderr,
+            )
+            sys.exit(1)
+        fork_full_name = head_repo["full_name"]
+        fork_clone_url = head_repo["clone_url"]
+        remote_name = f"pr-{pr['number']}"
+        local_branch = f"pr-{pr['number']}-{branch_name}"
+        print(f"PR head is in fork {fork_full_name}; fetching as remote '{remote_name}'...")
+        subprocess.run(["git", "remote", "remove", remote_name], capture_output=True, check=False)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)